src/style.css

```diff
@@ -0,0 +1,148 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  background-color: #fff;
+}
+
+.product-card__img {
+  height: 134px;
+  width: 160px;
+
+  margin: 32px 19px 0;
+}
+
+.product-card__item-title {
+  margin: 40px 16px 4px;
+  font-size: 12px;
+  font-family: inherit;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.product-card__item-code {
+  margin: 0 0 0 16px;
+
+  font-size: 10px;
+  font-family: inherit;
+  font-weight: 400;
+  color: #616070;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+
+  margin: 0 16px 16px;
+}
+
+.button {
+  display: flex;
+}
+
+.review__stars {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+
+  height: 16px;
+}
+
+.review {
+  display: flex;
+  justify-content: space-between;
+
+  margin: 16px 16px 24px;
```

Comment:
`margin: 16px 16px 24px;`
You can't set margins to BAM block.
Also use only margin-top or margin-bottom;